h264decoder.cpp

```diff
@@ -113,13 +127,13 @@ const AVFrame& ConverterRGB24::convert(const AVFrame &frame, ubyte* out_rgb)
   
   context = sws_getCachedContext(context, 
                                  w, h, (AVPixelFormat)pix_fmt, 
-                                 w, h, PIX_FMT_RGB24, SWS_BILINEAR, 
+                                 w, h, AV_PIX_FMT_RGB24, SWS_BILINEAR,
                                  nullptr, nullptr, nullptr);
   if (!context)
     throw H264DecodeFailure("cannot allocate context");
   
   // Setup framergb with out_rgb as external buffer. Also say that we want RGB24 output.
-  avpicture_fill((AVPicture*)framergb, out_rgb, PIX_FMT_RGB24, w, h);
+  av_image_fill_arrays(framergb->data, framergb->linesize, out_rgb, AV_PIX_FMT_RGB24, w, h, 1);
```

Comment:
No version check and conditional compilation required here?

Reply:
This works with the `libav` version released in 2016, which ships with Ubuntu 16.04. That seems like a reasonable amount of backwards-compatibility to me already, so I would not add an extra case here.